src/app/my/page.tsx

```
@@ -41,8 +43,16 @@ export default function ProfilePage() {
   }
 
   useEffect(() => {
-    loadUserInfo()
-  }, [userId])
+    if (userId !== null) {
+      setIsReady(true);
+    }
+  }, [userId]);
+
+  useEffect(() => {
+    if (isReady) {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: loadUserInfo

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)